active/RadialGradientBrush/RadialGradientBrush.md

```diff
@@ -38,26 +21,28 @@ Notable inconsistencies with the composition API include:
 1. Usage of [Point](https://docs.microsoft.com/uwp/api/windows.foundation.point) rather than [Vector2](https://docs.microsoft.com/uwp/api/windows.foundation.numerics.vector2)/[float2](https://docs.microsoft.com/windows/win32/numerics_h/float2-structure)
 2. Usage of [GradientStop](https://docs.microsoft.com/uwp/api/windows.ui.xaml.media.gradientstop) rather than [CompositionColorGradientStop](https://docs.microsoft.com/uwp/api/windows.ui.composition.compositioncolorgradientstop)
 
-The main reason for the above is to enable Xaml markup parsing for brushes defined in markup.
+The main reason for the above is to enable Xaml markup parsing for brushes defined in markup and consistency with the rest of the Xaml API.
```

Comment:
We've started using Numerics types for new Xaml properties against precedent (e.g. `UIElement.Translation` etc.) so we already have some inconsistency